ssstar/src/extract.rs

```diff
@@ -56,7 +56,7 @@ pub enum SourceArchive {
     File(PathBuf),
 
     /// Read the tar archive to some arbitrary [`std::io::Read`] impl.
-    Reader(Box<dyn Read + Send>),
+    Reader(Box<dyn Read + Send + Sync>),
```

Comment:
But this part is part of the public API, which makes this a breaking change.  So when you make a release of `ssstar`, use `minor` and not `patch`.

Reply:
I modified the changelog to reflect the changes. Will I need to create a new `## [Unreleased]` section in the changelog after the release of the `0.4.0` version?

Reply:
Yes, the process for making a release includes updating the changelog, so that there's a new `Unreleased` section and the current unreleased section becomes the section for the newly released version.  Right now only repo admins can perform a release because it involves pushing directly to `master`.  DM me on Slack when you're ready to do this and I'll kick it off for you.  Make sure the `CHANGELOG` is in place before you merge this.